src/PowerShellEditorServices.Hosting/Commands/StartEditorServicesCommand.cs

```diff
@@ -358,6 +359,7 @@ private EditorServicesConfig CreateConfigObject()
                 AdditionalModules = AdditionalModules,
                 LanguageServiceTransport = GetLanguageServiceTransport(),
                 DebugServiceTransport = GetDebugServiceTransport(),
+                LanguageMode = Runspace.DefaultRunspace.SessionStateProxy.LanguageMode,
```

Comment:
```suggestion
                LanguageMode = Runspace.DefaultRunspace.LanguageMode,
```

Reply:
That is on `RunspaceBase`:

https://github.com/PowerShell/PowerShell/blob/b1e998046e12ebe5da9dee479f20d479aa2256d7/src/System.Management.Automation/engine/hostifaces/ConnectionBase.cs#L1318-L1353

and this type is a `Runspace`.

Do you want me to cast?

Reply:
Oh, never mind then.